src/ground_truth_nsrts.py

```diff
@@ -1029,14 +1036,15 @@ def moveto_sampler(state: State, goal: Set[GroundAtom],
         }
         add_effects = {
             LiftedAtom(NextToBox, [robot, targetbox]),
-            LiftedAtom(NextTo, [robot, obj])
+            LiftedAtom(NextTo, [robot, obj]),
+            LiftedAtom(NotOnTable, [obj])
         }
         delete_effects = {
             LiftedAtom(NextToTable, [robot]),
             LiftedAtom(OnTable, [obj])
         }
         # Moving could have us end up NextTo other objects.
-        side_predicates = {NextTo, NextToBox, NextToShelf}
+        side_predicates = {NextTo}
```

Comment:
i feel like we should commit this change to master -- MoveToBox shouldn't have NextToBox or NextToShelf as side predicates right?